src/uct/ib/base/ib_md.c

```diff
@@ -103,6 +103,14 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "Use indirect atomic\n",
      ucs_offsetof(uct_ib_md_config_t, ext.enable_indirect_atomic), UCS_CONFIG_TYPE_BOOL},
 
+    {"GID_INDEX", "0",
+     "Port GID index to use.",
+     ucs_offsetof(uct_ib_md_config_t, ext.gid_index), UCS_CONFIG_TYPE_UINT},
+
+    {"SUBNET_PREFIX", "",
+     "Filter IB ports by subnet_prefix (no filter by default)",
```

Comment:
"Infiniband subnet prefix to filter ports by, empty means no filter. Relevant for IB link layer only\n"
"For example a filter for the default subnet prefix can be specified as: fe80:0:0:0"